app/assets/stylesheets/employee-page.scss

```diff
@@ -0,0 +1,102 @@
+strong {
+	font-weight: bold; 
+}
+
+em {
+	font-style: italic; 
+}
+
+table {
+	background: #f5f5f5;
+	border-collapse: separate;
+	box-shadow: inset 0 1px 0 #fff;
+	font-size: 12px;
+	line-height: 24px;
+	// margin: 30px auto;
```

Comment:
Get into a habit of removing commented out things before committing and pushing.